pkg/fileutil/fileutil.go

```diff
@@ -167,6 +168,11 @@ func CopyTree(src, dest string, uidRange *uid.UidRange) error {
 		return nil
 	}
 
+	fis, err := ioutil.ReadDir(dest)
```

Comment:
If the destination is an empty directory, `CopyTree` fails and I want to be able to call it whether the destination exists (and it's empty) or not.